apps/confidential/confidential.go

```diff
@@ -740,12 +739,11 @@ func (cca Client) AcquireTokenOnBehalfOf(ctx context.Context, userAssertion stri
 }
 
 // Account gets the account in the token cache with the specified homeAccountID.
-func (cca Client) Account(homeAccountID string) Account {
-	return cca.base.Account(homeAccountID)
+func (cca Client) Account(ctx context.Context, homeAccountID string) (Account, error) {
```

Comment:
Consider naming the id `accountId` instead of `homeAccountId`. The concept of "home" implies tenancy, which is only in AAD. ADFS doesn't have this. And as we expand to other IdPs, they likely won't have this either.